README.md

```diff
@@ -2,44 +2,61 @@
 
-1. Reliability. It's built upon Estimote Monitoring, Estimote's algorithm for reliable enter/exit reporting.
-2. No need to operate on abstract identifiers, or Proximity UUID, Major, Minor triplets. Estimote Proximity SDK lets you define zones by setting predicates for human-readable JSONs.
-3. You can define multiple zones for a single beacon, i.e. at different ranges.
-4. Cloud-backed grouping. When you change your mind and want to replace one beacon with another, all you need to do is reassign JSON attachments in Estimote Cloud. You don't even need to connect to the beacon!
+1. **Tag-based identification** - define and monitor zones with simple tags. 
+2. **Multiple zones per beacon** - set up more than one enter/exit zone per a single beacon 
+3. **Software-defined range** - allows you to define enter/exit trigger range. 
+4. **Cloud-based grouping using tags** - add, remove, and replace beacons without having to modify your source code - just modify tags in Estimote Cloud. All changes will be applied automatically. 
+
+# Table of Contents
+
+* [Tag-based identification](#tag-based-identification)
+* [Installation](#installation)
+* [How to use Proximity SDK in your app](#how-to-use-proximity-sdk-in-your-app)
+* [Location permissions](#location-permissions)
+* [Background support](#background-support)
+* [Additional features](#additional-features)
+* [Helpfull stuff](#helpful-stuff)
+* [Example app](#example-app)
+* [Your feedback and questions](#your-feedback-and-questions)
+* [Changelog](#changelog)
+
+# Tag-based identification
+
+Details of each of your Estimote devices are available in Estimote Cloud. Your beacons have unique identifiers, but memorising  all of them can be challenging. This is why Estimote Proximity SDK uses tag-based identification.
+
+>As our SDK is still in version `0.x.x`, we're constantly modifying our API according to your feedback. Our latest iteration has evoloved our SDK to be based on simple tags, backed up with attachments as an optional additional information. From the version `0.6.0`, the method `.forAttachmentKeyAndValue(...)` is deprecated - please use `.forTag(...)` instead.
+
+![Proximity zones based on tags](/images/tags.png)
+
+# Installation
 
 ## Requirements
 
-- One or more [Estimote Proximity or Location Beacon](https://estimote.com/products/) with the `Estimote Location` packet advertising enabled. 
+- One or more [Estimote Proximity or Location Beacon](https://estimote.com/products/) with `Estimote Monitoring` enabled.
 - An Android device with Bluetooth Low Energy support. We suggest using Android 5.0+ (Lollipop or newer). 
 - An account in [Estimote Cloud](https://cloud.estimote.com/#/)
-## Installation
+
+## Gradle
 
 Add this line to your `build.gradle` file:
 ```Gradle
 implementation 'com.estimote:proximity-sdk:0.5.1'
 ```
 > If you are using Gradle version below `3.0.0` then you should use `compile` instead of `implementation`.
 
```

Comment:
Or use our [Example app](#example-app) to download a ready, pre-integrated demo.